test/stat/stat_test.go

```diff
@@ -78,11 +78,12 @@ func TestCliStatForLinkerdNamespace(t *testing.T) {
 				"linkerd-prometheus":     "1/1",
 				"linkerd-proxy-injector": "1/1",
 				"linkerd-sp-validator":   "1/1",
+				"linkerd-tap":            "1/1",
 				"linkerd-web":            "1/1",
 			},
 		},
 		{
-			args: []string{"stat", "po", "-n", TestHelper.GetLinkerdNamespace(), "--from", "deploy/linkerd-controller"},
+			args: []string{"stat", "po", "-n", TestHelper.GetLinkerdNamespace(), prometheusPod, "--from", "po/" + controllerPod},
```

Comment:
I'm not sure what's going on here, why is the prometheus pod added to this command?

Reply:
This test was created to test that the controller was sending traffic to Prometheus initially but now, with the Linkerd tap pod in the mix, this test fails because the controller sends traffic to both Prometheus and linkerd tap. The traffic to the Linkerd tap pod is dependent on whether other tests were exercising the Tap API, which made this test brittle because we would sometimes see the stat command display stats for the tap pod and other times it wouldn't